src/mobile/src/ui/views/wallet/WalletResetRequirePassword.js

```diff
@@ -90,6 +90,7 @@ class WalletResetRequirePassword extends Component {
 
     componentWillUnmount() {
         BackHandler.removeEventListener('hardwareBackPress');
+        delete this.state.password;
```

Comment:
What does this `delete` statement add here? The component is unmounting anyway.

Reply:
It seems to reduce the instances in memory. Perhaps unmounting does not always delete all state variables.